app/assets/javascripts/nilavu/controllers/topic-predeploy.js.es6

```diff
@@ -178,6 +178,8 @@ export default Ember.Controller.extend({
     },
 
     refreshNotifications() {
+        const category = this.get("model.tosca_type").split(".")[1] == "container" ? "container" : "vm";
```

Comment:
@vinomca-megam  As you have a static field in `build-topic-route` shall be a good idea to reference like `BuildTopicRoute.CONTAINER`